Services/COPage/classes/class.ilPageObjectGUI.php

```diff
@@ -37,6 +37,7 @@ class ilPageObjectGUI
     public const PREVIEW = "preview";
     public const OFFLINE = "offline";
     public const PRINTING = "print";
+    protected \ILIAS\TestQuestionPool\QuestionInfoService $questioninfo;
```

Comment:
This seems a bit odd here. 

I assume this code was misplaced and belongs into the `afterConstruct` Function of the `ilTestPageGUI` Class.